puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -5,12 +5,20 @@
   include zulip::nginx
   include zulip::supervisor
 
-  $web_packages = [
-    # Needed to access our database
-    "postgresql-client-${zulip::base::postgres_version}",
-    # Needed for Slack import
-    'unzip',
-  ]
+  if $::osfamily == 'debian' {
+    $web_packages = [
+      # This is not necessary on CentOS because $postgresql package already includes the client
+      # Needed to access our database
+      "postgresql-client-${zulip::base::postgres_version}",
+      # Needed for Slack import
+      "unzip",
```

Comment:
Because `unzip` is added for Slack import, in the `web_packages`, 
the line after, `zulip::safepackage { $web_packages: ensure => 'installed' }` becomes mandatory even on CentOS.